circle.yml

```diff
@@ -291,46 +290,13 @@ commands:
                   no_output_timeout: '3m'
                   command: |
                       node --version
-                      mkdir test
-                      cd test
-                      echo "Initializing test project"
-                      npx @bahmutov/cly init --cypress-version << parameters.cypressVersion >>
+                      cd test-project
```

Comment:
this test is now run against a pre-scaffolded project, test-project
Gleb's cly project does not yet offer scaffolding for 10.0.0 and the CYPRESS_INTERNAL_FORCE_SCAFFOLD env has been removed from 10.0.0

Reply:
Since this test project will break next time we have a breaking change, it's probably a good idea to document generating this test project... maybe even a script, if you're feeling fancy

Reply:
I can add steps to re-scaffold, It's just a basic new project with the scaffolded tests. To properly script this we'd have to add back `CYPRESS_INTERNAL_FORCE_SCAFFOLD` or something like it that would allow us to scaffold a project via the CLI instead of through the UI (If there is a way to do that now, I didn't find it).

Reply:
Addressed in https://github.com/cypress-io/cypress-docker-images/pull/669/commits/a9da814af941e55ba4b99af755bc0a3f03af6b6e

Reply:
@mjhenkes `cypress/packages/example`'s build script does this, I guess we could clone that repo and use that to automate this (not saying to do this here, just spit-ballin')